### DISCUSSION TITLE
Jenkins 란?

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#43] [Q&A] [answered]
Jenkins 란?

**Comment by tjdud22** [comment, marked as answer]
일련의 자동화 작업의 순서들의 집합인 Pipeline을 통해 CI/CD 파이프라인을 구축합니다.
플러그인들을 잘 조합해서 돌아가게 하는 게 Pipeline이라고 할 수 있습니다.

**Comment by tjdud22** [comment]
소프트웨어 개발 시 지속적으로 통합 서비스를 제공하는 툴
